### ISSUE TITLE
[Bug] <Mangakawaii>: <not page found>

### ISSUE THREAD

**Issue by meliodasd** [tachiyomiorg/tachiyomi-extensions#5985]
## Device information
- Tachiyomi version: latest
- Android version: 10

## Source information
- Name: Mangakawaii 
- Extension version: latest

## Steps to reproduce
1. Click on a manga 
2. Click on a chapter 

You can't read or download a manga on this extension.

**Comment by happywillow0** [comment]
Hey @meliodasd 

Can you explain in a little more details exactly what you are doing and which buttons are you pressing that are not working? 

I just loaded the extension on my phone and while there are some things that are broken ("popular manga"), I am able to load chapters to read from the "Latest" screen.

**Comment by meliodasd** [comment]
Hi, Indeed the problem mentioned above is solved alone but the download does work

**Comment by happywillow0** [comment]
While testing, I noticed two items as well:

Some of the newest uploads that show up on the Latest screen brings you to a page that says: `En cours d'ajout...
Merci de repasser plus tard`. This may be why some chapters do not load. 

I also noticed at one point the server wasn't giving me the images. I'm not sure if they are having issues or are actively changing their site's software. It is working again now for some older chapters.
